Extensions/ServiceNow/Src/Tasks/CreateAndQueryChangeRequest/task.json

```diff
@@ -207,43 +315,88 @@
         {
           "RequestInputs": {
             "EndpointId": "$(ServiceNowConnection)",
-            "EndpointUrl": "$(endpoint.url)/api/now/import/x_mioms_azpipeline_change_request_import",
+            "EndpointUrl": "$(endpoint.url)/api/x_mioms_azpipeline/change_request_import",
```

Comment:
Why are we changing this. This means that the task will not work with old SNOW apps.

Reply:
Yes Aseem, This is a major version change. URL is changed as I am using now Scripted REST api to insert change request records.